### DISCUSSION TITLE
slash commands should show white-colored name

### DISCUSSION THREAD

**Discussion by tatitatai** [ppy/osu#26048] [Ideas]
![osu!_2023-12-19_18-34-10](https://github.com/ppy/osu/assets/32525896/5342afa3-68fd-4260-bf26-68f2c53aa4c2)

Looks like it got typed by hand atm. Addition: I don't know why but in stable it adds a * in front of your name, not sure if important lol

**Comment by bdach** [comment]
Why does this matter?

**Reply by Gabixel** [reply]
This might be useful in some cases. Right now, you can't tell the difference between a user-created message and an "automated" one (like with /np) in the case of a link (until you hover over it, of course). That's just a random example.
In general, it just looks nicer to have a different style for these messages, I guess?

![image](https://github.com/ppy/osu/assets/43073074/e4247796-7611-4c67-a0f0-8987c36ab99c)
![image](https://github.com/ppy/osu/assets/43073074/87cd0467-7f91-41fd-b21d-f23c205c5700)

Also web takes a similar approach:
![image](https://github.com/ppy/osu/assets/43073074/67e7114f-daea-4b03-99bd-5a94a08564ff)

**Reply by bdach** [reply]
If anything eventually the `/np` command will become an "action" message like `/me` commands on IRC and twitch. This will have the effect of colouring the message with the user's chat colour, i.e.:

![1703250269](https://github.com/ppy/osu/assets/20418176/581435c4-3c36-4e3a-91ae-f86efc72ed41)

This is probably what web does.

Does this answer the concern?
